### PR TITLE
Added Config Directory using Gitignore

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # "Describe what's new in this PR"

### How to Review

[//]: # 'The order in which to review files and what to expect when testing locally'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc"

### Next steps

[//]: # "What doesn't work yet, what's NOT in this PR/has to be done "

### Tests Performed, Edge Cases

[//]: # 'If you made changes/additions to `airtable.js` make sure you update/add'
[//]: # '[at least 1] corresponding unit tests in `airtable.spec.js`'

### Screenshots

[//]: # "Add screenshots!!! If you'd like"

CC: @dfangshuo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -111,4 +111,3 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
```
Comment (dfangshuo) [inline .gitignore:113]: jwt.key
jwt.key.pub

**Review by dfangshuo — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (aivantg) [reply .gitignore:113]: not technically necessary since the config folder is ignored, but good idea in case they somehow aren't

**Review by dfangshuo — APPROVED** [review]
